src/solvers/dgsem_tree/dg_1d_parabolic.jl

```diff
@@ -127,7 +130,7 @@ end
 function calc_volume_integral!(du, flux_viscous,
                                mesh::TreeMesh{1},
                                equations_parabolic::AbstractEquationsParabolic,
-                               dg::DGSEM, cache)
+                               dg::DGSEM, cache) where {uEltype <: Real}
```

Comment:
```suggestion
                               dg::DGSEM, cache)
```
This is no used here.